spec/unit/lib/web/page_spec.rb

```diff
@@ -178,8 +196,9 @@
       subject { described_class.expanded_url }
       before do
         allow(described_class).to receive(:url_template) { '/users' }
+        described_class.class_eval { root_url 'http://example.com' }
```

Comment:
same, in real life we will never do so, instead of that we will have:

```ruby
class Foo < Howitzer::Web::Page
  root_url 'https://google.com'
  url '/foo'
 end 
```